src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -663,12 +671,14 @@ public function fresh($with = [])
     /**
      * Eager load relations on the model.
      *
-     * @param  array|string  $relations
+     * @param  array|string|null  $relations
      * @return $this
      */
-    public function load($relations)
+    public function load($relations = null)
     {
-        if (is_string($relations)) {
+        if (null == $relations) {
```

Comment:
@ntzm I believe there is no syntactic reason for one operator over the other when comparing to `null` but I'll wait for someone more familiar to comment. Otherwise, this was not flagged by [the StyleCI analysis](https://styleci.io/analyses/XprjlV).